azure-pipelines.yml

```diff
@@ -4,29 +4,41 @@
 # https://docs.microsoft.com/azure/devops/pipelines/languages/docker
 
 trigger:
-- master
+  - master
 
 pr:
-- master
+  - master
```

Comment:
this should be set to `pr: none` to prevent pushing image to docker on pull request

Reply:
I will do this for pr specific yaml file